crescent/context.py

```diff
@@ -37,31 +37,49 @@
     from crescent.bot import Bot
 
 
-__all__: Sequence[str] = ("Context",)
+__all__: Sequence[str] = ("Context", "AutocompleteContext")
 
 
 @define
-class Context:
-    """Represents the context for interactions"""
+class BaseContext:
+    """Represents the context for command interactions"""
 
     interaction: CommandInteraction
+    """The interaction object."""
     app: Bot
+    """The client application."""
     application_id: Snowflake
+    """The ID for the client that this interaction belongs to."""
     type: int
+    """The type of the interaction."""
     token: str
+    """The token for the interaction."""
     id: Snowflake
+    """The ID of the interaction."""
     version: int
+    """Version of the interaction system this interaction is under."""
 
     channel_id: Snowflake
+    """The channel ID of the channel that the interaction was used in."""
```

Comment:
"The channel ID of the channel" feels redundant. Same for guild_id. Maybe just "The ID of the channel"?

Reply:
I felt like "The  ID of the channel" was confusing on what channel so I want to be more descriptive.